src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -503,44 +505,40 @@ namespace FanCoilUnits {
                 }
             }
 
-            errFlag = false;
-            ValidateComponent(state, fanCoil.FanType, fanCoil.FanName, errFlag, CurrentModuleObject);
-            if (errFlag) {
-                ShowContinueError(state, format("specified in {} = \"{}\".", CurrentModuleObject, fanCoil.Name));
+            fanCoil.fanType = static_cast<DataHVACGlobals::FanType>(getEnumValue(DataHVACGlobals::fanTypeNamesUC, Alphas(9)));
+            if (fanCoil.fanType == DataHVACGlobals::FanType::Invalid) {
```

Comment:
Fan type is a choice field and cannot be invalid at this point.

Reply:
Ok, can change this to an `assert`.

Reply:
Why? The assert will never fail.

    assert(fanCoil.fantype !=  DataHVACGlobals::FanType::Invalid);